Source/Atj/data/mvp_scenario.json

```diff
@@ -642,6 +807,16 @@
                 }
             ]
         },
+        {
+            "name": "zaks_breakfast_1_in_refrigerator_1_false_routine",
+            "tasks": [
+                {
+                    "type": "execute_action",
+                    "sync_time": "50.3",
```

Comment:
Why `50.3`? Round numbers will be easier to maintain unless theres a reason for the `0.3`